docs/javascript/v2/features/preview.mdx

```diff
@@ -57,9 +58,27 @@ tracks will be carry forwarded when user enters the room.
 
 ## Know Others in the room
 
-It's also possible to know about the room state and other peers in the room, if the permission to receive room state is given for the 
+It's also possible to know about the room state and other peers in the room, if the permission to receive room state is given for the
 role in dashboard. The peers present in the room can be known by using the `selectPeers` selector as given [here](../guides/useful-selectors#who-all-are-in-my-room).
 
+<Tabs id="peers" items={['Javascript', 'React']} />
+
+<Tab id='peers-0'>
+
+```js
+const peers = hmsStore.getState(selectPeers);
```

Comment:
we should use selectRemotePeers only here. Before joining, localpeer should not be part of the count